SwiftTweaks/TweakViewData.swift

```diff
@@ -31,6 +32,8 @@ internal enum TweakViewData {
 		case .double:
 			let clippedValue = clip(value as! Double, minimum as? Double, maximum as? Double)
 			self = .doubleTweak(value: clippedValue, defaultValue: defaultValue as! Double, min: minimum as? Double, max: maximum as? Double, stepSize: stepSize as? Double)
+        case .optionsList:
+            self = .optionsList(value: value as! StringOption, defaultValue: defaultValue as! StringOption, options: options!.map{ $0 as! StringOption })
```

Comment:
Just following the style in the SDK. I don't want it to introduce `guard` statements.